android/build.gradle

```diff
@@ -81,31 +74,3 @@ dependencies {
     // Inbrain Android SDK
     implementation 'com.github.inbrainai:sdk-android:[2.1.5, 2.2.0)'
```

Comment:
Hey @MusiikoAS  - let's update min version to the latest (2.1.12 instead of 2.1.5)

Reply:
Done